README.md

```diff
@@ -507,9 +509,9 @@ Content-Type: application/json
 [
     {
         "id": 1,
-        "name": "StandardLibrary - localadmin",
+        "name": "Jane's fabric elements",
         "type": "elements",
-        "items": {
+        "data": {
```

Comment:
why the change from `items` to `data` ? which is correct?

Reply:
`data` is correct. I only noticed when looking at the response!